File: Java/1020 - Idade em dias.java
import java.io.IOException;
import java.util.Scanner;

class Main {

	public static void main(String[] args) {

		Scanner read = new Scanner(System.in);

		int anos, meses, dias;

		dias = read.nextInt();

		anos = dias/365;
		meses = dias%365/30;
		dias = dias%365%30;

		System.out.println(anos + " ano(s)");
		System.out.println(meses + " mes(es)");
		System.out.println(dias + " dia(s)");
	}
}

File: Lua/1020 - Idade em dias.lua
function arredonda(info)

	info = tostring(info)
	number = ""
	quit = 0

	for i=1, string.len(info) do
		char = string.sub(info, i, i)

		if char ~= "." and quit == 0 then
			number = number .. char
		else
			quit = 1
		end
	end

	return number

end

dia = io.read()

ano = arredonda(dia/365)
mes = arredonda((dia%365)/30)
dia = arredonda((dia%365)%30)

print(ano .. "ano(s)")
print(mes .. "mes(es)")
print(dia .. "dia(s)")